### ISSUE TITLE
can we read an  SSE-C encrypted file in S3 ?

### ISSUE THREAD

**Issue by fcomte** [BlazingDB/blazingsql#993]
Hey,

I would like to use SSE-C to read file from S3 with a customer encryption key. Is it possible ?

**Comment by felipeblazing** [comment]
https://docs.blazingdb.com/docs/s3 shows how you can configure a bucket to set the encryption type for the bucket you will be reading.

**Comment by fcomte** [comment]
I don't think we can configure SSE-C because there is no encryption_key parameter.
SSE-C need a 256 bits key from the client.
